R/check_dhis2_attributes-helpers.R

```diff
@@ -0,0 +1,248 @@
+#' Get the dataset identifiers and names
```

Comment:
Function renaming _suggestion_ for the functions in this file

|New|Current
|-|-|
| get_datasets | `get_data_sets` 
| get_dataset | `get_relevant_dataset` 
| get_organisation_units | get_organisation_units (unchanged)
| get_organisation_unit | `get_relevant_organisation_unit` 
| get_data_elt_group  | `get_relevant_data_elt_group`  
| get_data_elements | get_data_elements (unchanged)

Reply:
These functions have been merged into 1 major function: `dhis2_get_relevant_attributes()`.